src/auth/routes.ts

```diff
@@ -32,6 +33,8 @@ router.get(gitea_redirect, passport.authenticate('gitea'));
 router.get(gitea_callback, passport.authenticate('gitea', passportOptions), authenticate, redirectToUi);
 router.get(google_redirect, passport.authenticate('google', googleOptions));
 router.get(google_callback, passport.authenticate('google', passportOptions), authenticate, redirectToUi);
+router.get(saml_redirect, passport.authenticate('saml', oidOptions));
```

Comment:
The `oidOptions ` doesn't exist anymore after you've removed OIDC. You can change this to `passportOptions `.